node/core/approval-voting/src/import.rs

```diff
@@ -447,6 +482,7 @@ pub(crate) async fn handle_new_head<Context, B: Backend>(
 		imported_blocks = imported_blocks_and_info.len(),
 		"Inserting imported blocks into database"
 	);
+	let mut db_insertion_span = handle_new_head_span.child("block-db-insertion");
```

Comment:
This function doesn't really write to DB. This happens at the parent scope. We use an overlay to accumulate writes and do the actual writing here: https://github.com/paritytech/polkadot/blob/master/node/core/approval-voting/src/lib.rs#L883 . Note this is already covered by metrics, but we could still use the span as it will be relevant.

Reply:
Thanks, will move this accordingly.